app/controllers/concerns/search_aggregator.rb

```diff
@@ -36,8 +36,8 @@ def aggregate_results(query, results)
       format_search_result(group, all_ontologies)
     end
 
-    if federation_enabled?
-      search_results = merge_sort_federated_results(query, search_results) if federation_enabled?
+    if federation_portals_enabled?
+      search_results = merge_sort_federated_results(query, search_results) if federation_portals_enabled?
```

Comment:
no need of this second check here,

Reply:
Done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/780/commits/ab9f5b4e1d42adc165bbf5a92bf9dd49e102bf35